src/systempolicy/systemPolicy.go

```diff
@@ -1464,6 +1464,17 @@ func StartSystemLogRcvr() {
 	for {
 		if cfg.GetCfgSystemLogFrom() == "kubearmor" {
 			url := cluster.GetKubearmorRelayURL()
+			if url == "" {
+				log.Error().Msg("kubearmor-relay url not found, retrying...")
+				for i := 0; i < 6; i++ {
```

Comment:
Why is there a hardcoded value of  6?

Reply:
it's trying connecting to kubeamor-relay, 6 is the max number of tries. We have 10 sec sleep, so trying 6 times gives us a minute of sleep before we return.

Reply:
@Ankurk99 Can you set a constant for this in types/constants.go and read/use it from there.

Reply:
Or.. Is it OK to read the retry timeout and no.of.times as a configuration from config file and use it here ?
@nyrahul WDYT ?